modules/accumulo/src/main/java/org/apache/fluo/accumulo/util/ColumnConstants.java

```diff
@@ -26,6 +26,7 @@
   public static final long TX_DONE_PREFIX = 0x6000000000000000L;
   public static final long WRITE_PREFIX = 0x4000000000000000L;
   public static final long DEL_LOCK_PREFIX = 0x2000000000000000L;
+  public static final long RLOCK_PREFIX = 0x0000000000000000L;
```

Comment:
It has a prefix of `000` binary.   Luckily this was open and sorted exactly where I needed it to.  There is one prefix left open `100` binary and I think this would sort last.